package.json

```diff
@@ -57,5 +57,9 @@
     "*.{ts,tsx,json,scss}": "prettier --write",
     "{shared,backend/core,backend/push,frontend}/**/*.{ts,tsx}": "npm run lint:js",
     "frontend/**/*.scss": "npm run lint:css"
+  },
+  "dependencies": {
+    "@types/react-notifications-component": "^3.1.1",
+    "react-notifications-component": "^4.0.1"
```

Comment:
react-notifications-component must be installed to frontend only

types must be installed as dev dependency

Reply:
Done!